config/src/test/java/org/springframework/security/config/annotation/web/configurers/saml2/Saml2LogoutConfigurerTests.java

```diff
@@ -346,6 +349,17 @@ public void saml2LogoutResponseWhenCustomLogoutResponseHandlerThenUses() throws
 		verify(getBean(Saml2LogoutResponseValidator.class)).validate(any());
 	}
 
+	@Test
+	public void saml2LogoutWhenPostProcessedToLogoutWithHttpGetThenLogsOutAndSendsLogoutRequest() throws Exception {
```

Comment:
I'd recommend `saml2LogoutWhenLogoutGetThenLogsOutAndSendsLogoutRequest`.

The reason is that how logout-get is not important to the test.